source/documentation/style-rules/index.liquid

```diff
@@ -40,38 +31,7 @@ selectors over and over again, you can write one style rules inside another.
 Sass will automatically combine the outer rule's selector with the inner rule's.
 {% endmarkdown %}
 
-{% codeExample 'nesting' %}
-nav {
-  ul {
-    margin: 0;
-    padding: 0;
-    list-style: none;
-  }
-
-  li { display: inline-block; }
-
-  a {
-    display: block;
-    padding: 6px 12px;
-    text-decoration: none;
-  }
-}
-===
-nav
-  ul
-    margin: 0
-    padding: 0
-    list-style: none
-
-  li
-    display: inline-block
-
-  a
-    display: block
-    padding: 6px 12px
-    text-decoration: none
-{% endcodeExample %}
-
+{% render 'code-snippets/example-nesting' %}
```

Comment:
Ah yes I had left this one in the file since the snippet wasn't called or used anywhere else

Reply:
Ah, makes sense. Not a big deal either way, but maybe there's an advantage to having all the examples in one place?